tests/boot/snapper_rollback.pm

```diff
@@ -30,7 +30,7 @@ sub run {
     script_run("cat /etc/os-release", 0);
     # rollback
     script_run("snapper rollback -d rollback-before-migration");
```

Comment:
Base the failed case. The timeout should add on "snapper rollback -d rollback...".

Reply:
The 'snapper rollback -d' just to show info, and test doesn't assert it so we don't need to increase timeout for 'snapper rollback -d'.